SQL/0000-00-03-ConfigTables.sql

```diff
@@ -256,3 +257,4 @@ INSERT INTO Config (ConfigID, Value) SELECT ID, 't1'    FROM ConfigSettings WHER
 INSERT INTO Config (ConfigID, Value) SELECT ID, 't2'    FROM ConfigSettings WHERE Name="modalities_to_deface";
 INSERT INTO Config (ConfigID, Value) SELECT ID, 'pd'    FROM ConfigSettings WHERE Name="modalities_to_deface";
 INSERT INTO Config (ConfigID, Value) SELECT ID, 'false'  FROM ConfigSettings WHERE Name="usePwnedPasswordsAPI";
+INSERT INTO Config (ConfigID, Value) SELECT ID, 'loris-dev@bic.mni.mcgill.ca'  FROM ConfigSettings WHERE Name="adminContactEmail";
```

Comment:
No way am I merging anything with that as a default email address.

Reply:
Fair enough. Do you have an alternate suggestion?

Reply:
`''`

Reply:
Done